src/modules/tracker.js

```diff
@@ -415,6 +417,18 @@ class Tracker {
       const bodyParams = {};
       const { items, revenue, order_id, section } = parameters;
 
+      if (order_id) {
+        // Don't send another purchase event if we have already tracked the order
+        if (helpers.checkOrderId(purchaseEventStorageKey, order_id)) {
```

Comment:
Might make sense to have the key in the helper file.  Also this function name is a bit weird ... maybe
* `helpers.hasOrderIdRecord(order_id)`
* `helpers.addOrderIdRecord(order_id)`